src/System/v2/Builders/EntriesBuilder.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\System\v2\Builders;
```

Comment:
why v2 ? and why do you need this Builder ?

Reply:
I thought "v2" is the proper way of calling these folders because they use base uri which ends like "/v2", so that can be some version name etc. I'll remove it.

The unneeded namespace is my fault.